### PR TITLE
feat: parse payload on multipart/form-data request

### LINKED ISSUE #7: Parse payload on multipart/form-data
We need to enable parsing of `multipart/form-data` uploads.

### PR DESCRIPTION
**Pull request by jo**
Closes #7

### CODE REVIEW

**Comment by dafortune** [comment]
Thanks so much! Great update! Could you add a quick test for this improvement? I have very little time this week. Otherwise I could probably do next week and merge

If you want, you can use https://github.com/daf-spr/hapi-qs/blob/master/tests/features/parse-payload.test.js as reference. Thanks!

**Comment by jo** [comment]
Hi @daf-spr,

I changed the test you mentioned to test against both form data types.

File: tests/features/parse-payload.test.js [hunk 1]
```diff
@@ -5,17 +5,22 @@ const rp = require('request-promise');
...
-        return rp({
+        var requestOptions = {
           method: method,
           uri: 'http://localhost:' + config.port,
-          form: form,
           json: json
-        })
+        };
+        if (isMultipart) {
+          requestOptions.formData = form
```
Comment (dafortune) [inline tests/features/parse-payload.test.js]: Missing semicolon

**Comment by dafortune** [comment]
Great! Thanks, just a minor detail (a couple of missing semicolons) and it is good to go.

**Comment by jo** [comment]
I added the two missing semicolons.

It would be great to include a linter which runs as part of the test. I recommend [standard](https://www.npmjs.com/package/standard).

**Comment by dafortune** [comment]
Great! Thanks. Will create a separate issue to add a linter.

**Pull request merged**